.github/workflows/sagemaker_workflow.yml

```diff
@@ -6,7 +6,42 @@ on:
       - master
 
 jobs:
+  build_and_push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Set MLFLOW_TRACKING_URI environment variable in Dockerfile
+        run: |
+          sed -i 's|ENV MLFLOW_TRACKING_URI=.*|ENV MLFLOW_TRACKING_URI=s3://${{ secrets.AWS_S3_BUCKET_NAME}}/models/mlflow-runs|' \
```

Comment:
MLFLOW_TRACKING_URI is meant to contain a server URI and PORT instead of an s3 path.
This env variable will be set later when the mlflow tracking server will be setup